### PR TITLE
Fraschetti - Add wallet-get-address-details-contract-test

### LINKED ISSUE #18349: Wallet - Add Contract test for RPC endpoint - wallet_getAddressDetails
As part of the wallet flows we want to add contract tests for all Status Go endpoints used by the wallet to help avoid the Go team adding breaking changes going unnoticed on the mobile client.

Add an integration test for the RPC request -
"wallet_getAddressDetails" used in src/status_im/contexts/wallet/events.cljs

For more information/examples on integration tests see this recent pr: https://github.com/status-im/status-mobile/pull/18304

### PR DESCRIPTION
**Pull request by FFFra**
fixes #18349
Epic #18587

Summary
This pull request introduces comprehensive tests for the `wallet_getAddressDetails` endpoint, ensuring its robustness and reliability in our wallet functionality.

Steps to test
run `make test-contracts`

status: ready!

### CODE REVIEW

**Comment by FFFra** [comment]
My RPC Calls returns:
` {:address "0xeefb13c7d42efcc655e528da6d6f7bbcf9a2251d", :path "", :hasActivity true, :alreadyCreated false}`
Its expected the `path` to be empty?

File: src/tests/contract_test/wallet_test.cljs [hunk 1]
```diff
@@ -65,3 +65,19 @@
     (fn []
       (p/let [response (contract-utils/call-rpc "wallet_getEthereumChains")]
         (assert-ethereum-chains response)))))
+
+(defn assert-address-details
+  [result]
+  (is (contains? result :address))
+  (is (contains? result :path))
+  (is (true? (:hasActivity result)))
```
Comment (smohamedjavid) [inline src/tests/contract_test/wallet_test.cljs]: `status-go` doesn't actually check whether there is an activity in that address instead it checks whether it has any balance or not.
If the address (`test.eth`) depletes the balance, this test will fail.

I would suggest checking whether the key is present in the result or not.
OR we can check whether this value is boolean (using the `boolean?` method).

Comment (FFFra) [reply src/tests/contract_test/wallet_test.cljs]: Tks! I was not aware. Gonna do this change!

**Review by ulisesmac — APPROVED** [review]
Thanks for these PRs @FFFra  :100:

**Review by smohamedjavid — APPROVED** [review]
🚀

**Pull request merged**